### PR TITLE
Dt 54 fix undo redo edit video

### PR DESCRIPTION
**Pull request by Duy-Tinh-Dev**

### CODE REVIEW

**Comment by kennytruong06** [comment]
oki

File: Views/EditVideo/index.tsx [hunk 1]
```diff
@@ -305,12 +305,12 @@ export default function EditVideo() {
             description: "",
         };
 
-        const templeDataSplit = dataSplit;
-        templeDataSplit[indexCurrentProgress].width = firstHalfPercentWidthProgress;
+        const templeDataSplit = JSON.parse(JSON.stringify(dataSplit));
         templeDataSplit[indexCurrentProgress].startTime = currentProgress.startTime;
```
Comment (lamxuanthinh) [inline Views/EditVideo/index.tsx:309]: Please check format code

**Comment by lamxuanthinh** [comment]
pass

**Comment by Giang-st1-00** [comment]
ok

**Pull request merged**